content/news/posts/2023/a-new-organization-for-the-open-source-project.md

```diff
@@ -0,0 +1,152 @@
+* Less is more. We strive not to overthink things!
+
+## Initial composition of Project Councils
+
+The initial composition of the project's Councils is largely based on the established roles from our previous structure. Current maintainers (including scoped maintainers) will automatically join the Technical Council. The rest of the Councils will be initially populated with the people currently performing similar roles.
+
+As we roll out the new structure out and Councils begin to function, new members may be invited, and inactive members may be removed.
+
+**Leadership Council**
+
+* Pablo Borowicz (@eternoendless) **– Project Lead**
+* Fabien Vallon (@fabienvallon)
+* Mathieu Ferment (@matks)
+* Robin Fischer (@Robin-Fischer-PS)
+
+**Technical Council**
+
+* Jonathan Lelièvre (@jolelievre) **– Lead**
+* Daniel Hlavacek (@Hlavtox)
+* Ibrahima Sow (@sowbiba)
+* Krystian Podemski (@kpodemski)
+* Mathieu Ferment (@matks)
+* Matthieu Rolland (@matthieu-rolland)
+* Pablo Borowicz (@eternoendless)
+* Pululu Kinanga André (@PululuK)
+* Thomas Baccelli (@atomiix)
+* Valentin Szczupak (@NeOMakinG)
+
+**Product Council**
+
+* Fabien Vallon (@fabienvallon) **– Lead**
+* Mateus Shirlaw (@MatShir)
+* Rosa Benouamer (@RosaBenouamer)
+* Zoé Finders (@Zoefinders)
+
+**Quality Council**
+
+* Robin Fischer (@Robin-Fischer-PS) **– Lead**
+* Boubker Bribri (@boubkerbribri)
+
+## Advantages of the new organization
+
+We think that this structure will provide many advantages:
+
+It will **facilitate decision-making**, thanks to a clearly-defined project structure, including the people responsible for each domain, that clears out any misunderstanding as to who is responsible for what decision (and especially who has the "final-final-final" word on it).
+
+It will **enable more people to participate in the project,** by opening decision-making roles that were previously closed to community members, including the creation of new lower-stakes roles that are easier to join to.
+
+It will **progressively accelerate decision-making,** by the means of simplified processes, and above all, more people to make them!
+
+All while **lowering the risk of project instability** due to diverging opinions.
+
+## Final words
+
+This new organization will progressively roll out in the coming weeks.
+
+Our objective is to accelerate time-to-decision to prevent contributors from getting stuck, and through that help more contributors succeed in participating in the project. Remember, **the more people contribute, the better PrestaShop becomes.**
+
+To that end, we intend to invite as many people in Councils (or delegate groups) as we can. The idea is to reach a point where if enough community members think a feature is interesting and are willing to contribute it, they will be able to self-organize to work on it, and Councils will be numerous enough to quickly react and provide guidance on the best way to include it. 
```

Comment:
```suggestion
To that end, we intend to invite as many people in Councils (or delegate groups) as we can. The idea is to reach a point where if enough community members think a feature is interesting and are willing to contribute to it, they will be able to self-organize to work on it. Councils will be numerous enough to quickly react and provide guidance on the best way to include it. 
```

Reply:
Yes I understand your point of view for the "to it", it didn't look right in the first reading but put in a technical context it does make sense. For the sentence split, the idea behind adding a dot was to shorten the long sentence and take some of the "and"s out to ease reading.

Reply:
Thanks Pablo, it's good for me.